src/app/create-suggested-dates/create-suggested-dates.component.html

```diff
@@ -507,11 +507,45 @@
             }
           </div>
         }
+        <div class="my-2 divider-single"></div>
+        <!-- description (optional) for this date entry -->
+        <div class="row">
+          <div
+            [ngClass]="{'has-error': getDescriptionByIndex(i).invalid
+              && (getDescriptionByIndex(i).dirty || getDescriptionByIndex(i).touched)}"
+            class="col form-group invalid-message-highlight"
+          >
+            <label data-id="descriptionLabel" for="suggested-date-description-{{i}}">
+              {{ 'poll.details.description.descriptionOptional' | translate }}
+            </label>
+            <div class="input-group">
+              <div class="input-group-prepend">
+                <div class="input-group-text input-group-divider">
+                  <img ngSrc="assets/description.svg" alt="Sprechblase" height="height" width="width">
+                </div>
+              </div>
+              <input
+                id="suggested-date-description-{{i}}"
+                data-id="descriptionInput"
+                class="form-control"
+                formControlName="description"
+                placeholder="{{ 'poll.details.description.description' | translate }}"
+                type="text"
+                aria-describedby="maxLength-description"
```

Comment:
müsste die ID nicht so 
```suggestion
                aria-describedby="description-too-long"
```
aussehen?